ldk/csharp/OliveHelpsLDK/Example/Program.cs

```diff
@@ -65,11 +65,10 @@ private void ClipboardStream()
             _clipboardStream = _services.Clipboard.Stream();
             Task.Run(async () =>
             {
-                while (await _clipboardStream.MoveNext())
+                await foreach (var clipboardContent in _clipboardStream.ToAsyncEnumerable())
```

Comment:
I like this way of doing it - gives a bit more flexibility. I could see folks being a little confused or having to use code completion to get to this point, but I think with a good example like this it's fine.